lib/templates/courant/courantbeheer.tpl

```diff
@@ -62,9 +62,10 @@
 		<input type="button" value="Voorbeeld" onclick="window.bbcode.CsrBBPreview('courantBericht', 'bewerkPreview');" />
 		{if $courant->magBeheren()}
 			<input type="button" value="Importeer agenda" onclick="window.courant.importAgenda('courantBericht');" />
+			<input type="button" value="Importeer sponsor" onclick="document.getElementById('courantBericht').value += '[img]https://csrdelft.nl/plaetjes/fotoalbum/Sponsoren/_resized/Projectkwadraat.jpg[/img]'" />
```

Comment:
Het logo kan denk ik beter niet in het fotoalbum, maar gewoon in https://csrdelft.nl/plaetjes/banners

Reply:
Misschien kunnen we ook een instelling maken voor de URL van het logo. Dan kan de sponsor worden gewijzigd zonder de code aan te passen.

Reply:
Dit is nu #464